src/main/java/com/celements/mailsender/CelMailSenderService.java

```diff
@@ -61,14 +67,11 @@ IMailObjectRole getNewMailObject() {
 
   @Override
   public String getEmailValidationRegex() {
-    // chars - a-z A-Z 0-9 œ à á â ã ä å æ ç è é ê ë ì í î ï ð ñ ò ó ô õ ö ø ù ú û ü ý þ ÿ
```

Comment:
Add this comment to EMAIL_VALIDATION_REGEX. It's much more readable than "\\w\\-\\u0153\\u00E0-\\u00F6\\u00F8-\\u00FF" if out want to no which characters are allowed.